mappings/net/minecraft/client/render/RenderPhase.mapping

```diff
@@ -70,15 +129,38 @@ CLASS net/minecraft/class_4668 net/minecraft/client/render/RenderPhase
 			ARG 2 y
 	CLASS class_4683 Texture
 		FIELD field_21397 id Ljava/util/Optional;
-		FIELD field_21398 bilinear Z
+		FIELD field_21398 blur Z
 		FIELD field_21399 mipmap Z
+		METHOD <init> (Lnet/minecraft/class_2960;ZZ)V
+			ARG 1 id
+			ARG 2 blur
+			ARG 3 mipmap
 	CLASS class_4684 Texturing
 	CLASS class_4685 Transparency
 	CLASS class_4686 WriteMaskState
-		FIELD field_21400 color Z
-		FIELD field_21401 depth Z
+		FIELD field_21400 writeColor Z
+		FIELD field_21401 writeDepth Z
```

Comment:
```suggestion
		FIELD field_21400 color Z
		FIELD field_21401 depth Z
```
Imo the rename is relatively poor. We are already writing in the state so `write` is redundant. Mojang using a redundant name doesn't mean we should too.